src/dataServices/dynamoDbHelper.ts

```diff
@@ -17,18 +17,37 @@ export default class DynamoDbHelper {
         this.dynamoDb = dynamoDb;
     }
 
-    async getMostRecentResource(
+    private async getMostRecentResources(
         resourceType: string,
         id: string,
         projectionExpression?: string,
-    ): Promise<GenericResponse> {
-        const params = DynamoDbParamBuilder.buildGetResourcesQueryParam(id, 1, projectionExpression);
+    ): Promise<ItemList> {
+        const params = DynamoDbParamBuilder.buildGetResourcesQueryParam(id, resourceType, 2, projectionExpression);

```

Comment:
can we pass in the `2` as a param; perhaps `size`?

Reply:
yup. updated